src/Raven.Server/Documents/Queries/Sorting/AlphaNumeric/AlphaNumericFieldComparator.cs

```diff
@@ -104,6 +104,9 @@ private AlphanumComparer()
 
             public unsafe struct AlphanumericStringComparisonState
             {
+                public char[] CurCharacters;
+                public int CurrentCharacterCharAmount;
```

Comment:
Do we have any usage of this?

Reply:
CurrentCharacterCharAmount - removed.
CurCharacters - is a buffer for chars when we looking for numbers.